### ISSUE TITLE
Impaired growth hormone secretory responses after growth hormone–releasing hormone challenge 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6507]
**Preferred term label:**
Impaired growth hormone secretory responses after growth hormone–releasing hormone challenge 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0031073

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/21651393/  2/2
OMIM:101800 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
